handlers/error.go

```diff
@@ -101,7 +104,7 @@ func ErrUnmarshal(err error, obj string) error {
 }
 
 func ErrEncoding(err error, obj string) error {
-	return errors.New(ErrEncodingCode, errors.Alert, []string{"Error encoding the : " + obj}, []string{err.Error()}, []string{"Unable to decode the : " + obj}, []string{})
+	return errors.New(ErrEncodingCode, errors.Alert, []string{"Error encoding the : " + obj}, []string{err.Error()}, []string{"Unable to encode the : " + obj}, []string{})
```

Comment:
Instead to appending the string, lets move to spperating then by `,`, like
```
return errors.New(ErrEncodingCode, errors.Alert, []string{"Error encoding the object", obj}, []string{err.Error()}, []string{"Unable to encode the object ", obj}, []string{})
```

This way the error utility tool would be able to detect and export the string for the docs, while also being able to see the object in meshery server.

`ErrUnmarshal` and `ErrMarshal` would also need some rework.

Reply:
Yeah okay I will do the rework.